llvm/lib/Analysis/MemoryProfileInfo.cpp

```diff
@@ -145,13 +146,6 @@ std::string llvm::memprof::getAllocTypeAttributeString(AllocationType Type) {
   llvm_unreachable("invalid alloc type");
 }
 
-static void addAllocTypeAttribute(LLVMContext &Ctx, CallBase *CI,
```

Comment:
There was only one call to this so I inlined it for simplicity and so we have easier access to the AllocTypeString for the remark.